play.pokemonshowdown.com/js/client.js

```diff
@@ -2555,6 +2555,10 @@ function toId() {
 			this.$el.html('<form><p style="white-space:pre-wrap;word-wrap:break-word">' + (data.htmlMessage || BattleLog.parseMessage(data.message)) + '</p><p class="buttonbar">' + (data.buttons || '<button type="button" name="close" class="button autofocus"><strong>OK</strong></button>') + '</p></form>').css('max-width', data.maxWidth || 480);
 		},
 
+		copyText: function (value, target) {
+			app.curRoom.copyText(value, target)
```

Comment:
```suggestion
			app.curRoom.copyText(value, target);
```
This is what I get for forgetting to approve github actions before reviewing the code.